src/main/java/de/tum/in/www1/artemis/service/metis/PostingService.java

```diff
@@ -229,4 +236,57 @@ else if (authorizationCheckService.isTeachingAssistantInCourse(postingCourse, po
     }
 
     protected abstract String getEntityName();
+
+    /**
+     * Gets the list of logins for users mentioned in a posting.
+     * Throws an exception, if a mentioned user is not part of the course.
+     *
+     * @param course         course of the posting
+     * @param postingContent content of the posting
+     * @return the list of logins of mentioned users
+     */
+    protected List<String> parseUserMentions(@NotNull Course course, String postingContent) {
```

Comment:
IntelliJ complains that you're never using the return value. Is that intended?
Also, do we have a unit test for this method? Seems like the tests are creating a few posts with mentions but we don't have an input-output test for this method, right?

Reply:
I let the method return the list of parsed mentions, since this is probably needed later, when we want to notify mentioned users. However, since it is not needed right now, I removed it.
I also added some unit tests for this method.